docs/install.rst

```diff
@@ -0,0 +1,352 @@
+about :ref:`howtos:clustering` and :ref:`howtos:going-into-production`. When
+operating a CrateDB cluster in production, `performance tuning`_ will also be
+of interest.
+
+.. _performance tuning: https://crate.io/docs/crate/howtos/en/latest/performance/
+
+.. NOTE::
+
+    To add additional CrateDB nodes to this kind of cluster in order to make it
+    form a multi-node cluster, you will need to remove the cluster state after
+    changing the configuration.
+
+.. NOTE::
+
+    If you are interested to follow the development more closely, please check
+    out the :ref:`reference:release_notes` for more specific information about
+    each CrateDB release. `Other releases of CrateDB`_ are also available.
+
+
+Prerequisites
+=============
+
+CrateDB requires a `Java virtual machine`_ to run.
+
+- Starting with CrateDB 4.2, a `Java virtual machine`_ is bundled with CrateDB
+  and no extra installation is necessary.
+
+- CrateDB version 3.0 to 4.1 required a separate Java installation, version
+  11 was the minimum requirement. Earlier versions required Java 8.
+  We recommend using OpenJDK_ on Linux Systems and `Oracle Java`_ on
+  Microsoft Windows and macOS.
+
+
+Basic setup
+===========
+
+This section of the documentation outlines how to use the release archives to
+install CrateDB.
+
+
+Automatic installation
+----------------------
+
+CrateDB has an effortless installer suitable for all `Unix-like`_ systems
+running ``bash``::
+
+    bash -c "$(curl -L https://try.crate.io/)"
+
+
+Manual installation
+-------------------
+
+This walkthrough is suitable to install and run CrateDB on both `Unix-like`_
+systems as well as on Microsoft Windows.
+
+#. Download the latest `CrateDB release archive`_. Please make sure to select
+   the right release archive matching your system.
+
+#. Once downloaded, extract the archive either using your favorite terminal or
+   command line shell or by using a GUI tool like `7-Zip`_::
+
+       tar -xzf crate-*.tar.gz
+
+#. On the terminal, change into the extracted ``crate`` directory::
+
+       cd crate-*
+
+#. Run a CrateDB single-node instance on the local network interface::
+
+       ./bin/crate
+
+#. In order to stop CrateDB again, use :kbd:`ctrl-c`.
+
+.. SEEALSO::
+
+      Consult the :ref:`reference:cli` documentation for further information
+      about the ``./bin/crate`` command.
+
+
+Notes about Microsoft Windows
+-----------------------------
+
+If you are installing CrateDB on a recent `Windows Server`_ edition, setting
+up the latest *Microsoft Visual C++ 2019 Redistributable* package is required.
+You can download it at `msvcrt x86-64`_, `msvcrt x86-32`_ or `msvcrt ARM64`_.
+
+Within the terminal, as a Windows user, the prompt after `starting PowerShell`_
+will look like this.
+
+.. code-block:: doscon
+
+    PS> ./bin/crate
+
+
+Debian and Ubuntu
+=================
+
+This section of the documentation outlines how to install CrateDB deb_ packages
+using the apt_ package manager.
+
```

Comment:
I think if we fold installation via package managers in the same documentation as the tarball installation we have to make it clear that this is an alternative way to install it. This should probably also be done at the start of the document - a sentence or something to point out that there are multiple ways to install it with links to the relevant sections.


I'm actually not sure if it is a good idea to merge this into the same document. I find it confusing to follow a documentation where you have sections which you need to skip as it's not always clear where you can continue. 

I think this ties into my earlier comments about the "follow up actions" that are already mentioned in the introduction.

Reply:
I have kind of the same opinion. So, either
a) split it again into different documents.
b) put the tarball installation at the bottom (as @proddata mentioned elsewhere, we should educate our users to better use "the real thing" instead of an ad hoc installation mode) and outline the difference within an introductory sentence outlining the different ways of installations, as you suggested.

Reply:
> I second that. However, I also support @proddata's idea that the package manager based installation should be advertised as the primary installation variant.

I think it makes a lot of things easier, especially upgrading to new versions.
I also think it is ok to have two separate instructions, just not 5